src/error.ts

```diff
@@ -520,8 +520,8 @@ export class MongoAWSError extends MongoRuntimeError {
    *
    * @public
    **/
-  constructor(message: string) {
-    super(message);
+  constructor(message: string, { cause }: { cause?: AnyError } = {}) {
+    super(message, { cause });
   }
```

Comment:
```suggestion
  constructor(message: string, options?: { cause?: Error }) {
    super(message, options);
  }
```

nit, no need to destructure just to create a new object. And `AnyError` is equivalent to `Error` here.